components/button.html

```diff
@@ -0,0 +1,171 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- Icon link -->
+    <link rel="icon" href="../assets/icon.png">
+    <!-- css style links -->
+    <link rel="stylesheet" href="../css/base_styles/reset.css">
+    <link rel="stylesheet" href="../css/base_styles/home.css">
+    <link rel="stylesheet" href="../css/component_styles/button.css">
+    <link rel="stylesheet" href="../css/component_styles/alert.css">
+    <link rel="stylesheet" href="../css/base_styles/introduction.css">
+    <link rel="stylesheet" href="../css/base_styles/media.css">
```

Comment:
You can import all your CSS into one file and attach a single file to your HTML files.